src/domUpdates.js

```diff
@@ -62,10 +62,26 @@ const renderGrid = () => {
   recipeGrid.innerHTML = createGridHTML(gridData);
 }
 
+const showRecipe = (recipeCard) => {
+  const recipeCardName = recipeCard.closest(".individual-recipe").querySelector("h2");
+  const thisRecipe = recipeData.find(recipe => recipe.name === recipeCardName);
```

Comment:
I think we may have an actual use case for an id here. Wouldn't it be more reliable to find a recipe with a matching id rather than matching name? What if somehow we get two recipes with the same name in the future? ( I know it is a super edge case) 

This would require changing some of Taranver's previous code to include an id of `singleRecipe.id` in line 28 of `createSingleRecipeHTML()` within the article tag:
- something like `<article class="individual-recipe" id="${singleRecipe.id}"> `

Once this change is made, we could actually just say `recipeCardId = recipeCard.closest('article').id` and find the id in line 67 instead of finding the name. 
- Keep in mind that within the find, when looking for the recipe's id you will need to use `.toString()` to match it to the id in the article since we can not match a number and string.

`thisRecipe` doesn't actually get used in this function but I know it will be necessary once we fill the card with the recipe information and I think this change could be a safe guard for our code

Reply:
Ooh great points - can we make a follow up card to refactor with this in mind?  Or we can add it as an item for when we actually start presenting that data on the recipe view in one of our cards we're about to work on after this merges @lauraguerra1 ?

Reply:
I agree. If our data is divided into unique ID's, those ID's should somehow reside in our HTML as well. Good catch @lauraguerra1 .

@mbenfowler I can do a card for refactoring my code to have an ID. This pull request seems fine as is.

Reply:
Yes, I think either of those works great, no need to do it within this specific PR